README.md

```diff
@@ -92,6 +92,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Padloc](https://github.com/padloc/padloc) - Modern, open source password manager for individuals and teams.
 - [Ghorbu Wallet](https://github.com/matthias-wright/ghorbu-wallet) - Cross-platform desktop HD wallet for Bitcoin.
 - [Stable Diffusion Buddy](https://github.com/breadthe/sd-buddy) - Desktop UI companion for the self-hosted Mac version of Stable Diffusion.
+- [Lettura](https://github.com/zhanglun/lettura) - Open-source feed reader for macOS, maybe for Windows/Linux in someday.
```

Comment:
```suggestion
- [Lettura](https://github.com/zhanglun/lettura) - Open-source feed reader for macOS.
```

When Linux and Windows are not yet supported it's a little misleading to advertise it using those words. Even though you explained it's not supported yet, that adds little information to readers while possibly confusing some.

Alternative just leave out all mention of OS support:
```suggestion
- [Lettura](https://github.com/zhanglun/lettura) - Open-source feed reader.
```